### DISCUSSION TITLE
Graphical GTK interface for TTS

### DISCUSSION THREAD

**Discussion by technout** [coqui-ai/TTS#2057] [General]
I follow this very interesting project for almost a year and decided to add some graphical interface for the simple user (which does not train his/her own model). It's just a small contribution to this cool project. And it is still in early beta stage, but should be working.
This graphical interface for Coqui TTS made in GTK3 and Python3 for the Linux (Ubuntu) platform.
You can find it on my Github: https://github.com/technout/tts_gtk

Preview:
![screen_tts_gtk_1](https://user-images.githubusercontent.com/20970468/194383129-7a301437-6733-4d4f-90d0-522b1b843b0b.png)

**Comment by lexkoro** [comment]
looks nice! :)

for a future feature I would suggest parsing this file https://github.com/coqui-ai/TTS/blob/dev/TTS/.models.json
and providing the available models in a dropdown list.

**Reply by technout** [reply]
Thanks, that is an interesting feature to add, i will look into that!

**Comment by p0p4k** [comment]
Very well made!

**Reply by technout** [reply]
Thank you! But it's still in early development.
